docs/config/lua/config/scroll_to_bottom_on_input.md

```diff
@@ -0,0 +1,3 @@
+# `scroll_to_bottom_on_input`
+
+When true (the default) scrolls to the bottom of the scrollback on input.
```

Comment:
I'd say:

> When `true` (the default), the viewport will automatically scroll to the bottom of the scrollback when there is input to the terminal so that you can see what you are typing.